lerna.json

```diff
@@ -7,8 +7,11 @@
     },
     "bootstrap": {
       "npmClientArgs": ["--no-package-lock"]
+    },
+    "version": {
+      "allowBranch": "master"
```

Comment:
Does this force only running version on `master`?

Reply:
yes